examples/convex_hull/src/main.cpp

```diff
@@ -14,7 +14,7 @@
 #include <fstream>
 #include <vector>
 
-#include "utils.h"
+#include "utils.hpp"
```

Comment:
It's covered by https://github.com/oneapi-src/oneDPL/pull/170/files#diff-324573e248045a8462e86fdd8ae69d302e832b699e32fdd201c67d4205973aa8R17 
You don't need to do it here

Reply:
Rolled back.